### ISSUE TITLE
No explanation what an eRFC is?

### ISSUE THREAD

**Issue by steffahn** [rust-lang/rustc-dev-guide#1451]
Took me a while to find out [what an eRFC is](https://github.com/rust-lang/rfcs/pull/2033#issuecomment-309057591), I believe such information should also be in the dev-guide, but it isn’t AFAICT.

**Comment by JohnTitor** [comment]
Maybe https://rustc-dev-guide.rust-lang.org/walkthrough.html#pre-rfc-and-rfc would be a good candidate to document it.

**Comment by tshepang** [comment]
it seems they are rarely used these days

**Comment by JohnTitor** [comment]
Yeah, but the explanation is still helpful when someone reads an old eRFC or related topic, e.g. https://rust-lang.github.io/rfcs/2318-custom-test-frameworks.html or https://github.com/rust-lang/rfcs/pull/2136.

**Comment by me-diru** [comment]
Hello, May I contribute to this?

**Comment by camelid** [comment]
Hey @me-diru, are you still planning to work on this issue? If not, I can take it over. If you want to work on it, let me know if you need any help!

**Comment by reddevilmidzy** [comment]
I'd like to work on this issue. 

From what I understand, the eRFC process isn't commonly used nowadays because it has become more efficient to iterate through experiments and gather feedback on the Nightly channel via the standard RFC process. Could you let me know if I've misunderstood anything, or if there are any additional resources I should check out?

**Comment by tshepang** [comment]
that sounds right, but I can't really confirm... maybe you can ask some people
